spec/lib/seeds_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Rails::Application do
+  it 'loads our seed data successfully' do
+    expect(Framework.count).to eq(0)
+
+    Rails.application.load_seed
+
+    expect(Framework.count).not_to eq(0)
```

Comment:
What do you think about asserting for the exact number of Frameworks we are attempting to create? I can imagine that we would have to update this test every time a framework was added or removed in future which might be tedious and this flexibility would help avoid. I'm wondering that if anything changed to the way frameworks were created we might be in the situation where we silently ended up with less frameworks than we thought, so long as at least 1 exists.

Reply:
I think that you're right that it might be tedious. I really just wanted to check that _something_ got imported into the database - primarily, to assure myself that when I called `load_seed` it was indeed targeting the test database and not development or something like that.